tutorials/vpc-private-gateway-autoconfiguration/index.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: Deploying Instances in a Private Network and exposing them using a Public Gateway
+description: This page shows how to deploy virtual Instances in a Private Network and expose them using a Public Gateway
+excerpt: |
+  VPC consists of Private Networks and Public Gateways to create an isolated network environment for your instances, without exposing them on the public Internet. 
+totalTime: PT20M
+  - network
+brands:
+  - vpc
+keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, network, virtual instance, private network, vpc, public gateway, virtual private cloud
+---
+
+## VPC - Overview
+
+VPC is a family of products providing network functionalities within the customer cloud. It consists in:
```

Comment:
Maybe take the text from the console here? VPC (Virtual Private Cloud) provides network functionalities for your Scaleway cloud. VPC products include Private Networks, enabling you to build a virtual L2 network between your instances, and the Public Gateway, enabling IP autoconfiguration of your Private Networks and their communication with the Internet.